packages/flutter/lib/src/animation/tween.dart

```diff
@@ -55,6 +55,16 @@ class _ChainedEvaluation<T> extends Animatable<T> {
 }
 
 /// A linear interpolation between a beginning and ending value.
+///
+/// [Tween] is useful if you want to map (or *interpolate*) from an input range to an output range.
```

Comment:
you don't interpolate from an input range to an output range, you interpolate across a range (from start to end). So the grammar here with the parenthetical is dubious.